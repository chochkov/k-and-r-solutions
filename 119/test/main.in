abcde123
fghij
klmno
p
