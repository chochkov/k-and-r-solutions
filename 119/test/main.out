321edcba
jihgf
onmlk
p
